docs/GOVERNANCE.md

```diff
@@ -0,0 +1,184 @@
+# Express.js Community Contributing Guide 1.0
```

Comment:
This was moved from here: https://github.com/expressjs/express/blob/master/Contributing.md